ir/lib/feedparser.py

```diff
@@ -130,7 +130,7 @@ def _l2bytes(l):
 #ACCEPTABLE_URI_SCHEMES = ()
 
 # ---------- required modules (should come with any Python distribution) ----------
-import cgi
+from .cgi import parse_header

```

Comment:
```suggestion
from . import cgi
```

`parse_header` is invoked as `cgi.parse_header`